src/index.html

```diff
@@ -6,7 +6,38 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="calendar calendar--start--tue calendar--month-lenght--31">
+    <div class="day day--1"></div>
+    <div class="day day--2"></div>
+    <div class="day day--3"></div>
```

Comment:
no needs to add modifiers to calendar day
```suggestion
    <div class="calendar__day"></div>
    <div class="calendar__day"></div>
    <div class="calendar__day"></div>
```

Reply:
Thanks! And tell me please to which tag should I bind the cycle with numbers in each cell?